app/views/items/_category_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>
+  <%= form_for Category.new, html: { class: 'form-inline' } do |f| %>
```

Comment:
haha! It was actually your suggestion of the presenters pattern that inspired me to write-up a more detailed description of the solution for Ruti! It's a great pattern but I was not satisfied with the blog post she linked so I kinda just wrote my own.